scrapers/SARJ-LLC.py

```diff
@@ -0,0 +1,141 @@
+import json
+import os
+import sys
+import re
+import requests
+from urllib.parse import urlparse, urlunparse, urlencode
+
+def scrape_url(url, type):
+    parsed = urlparse(url)
+
+    path = parsed.path.split('/')
+    baseUrl = f"{parsed.scheme}://{parsed.netloc}"
+    if type == 'scene':
+        index = path.index('movie')
+        scraped = scrape_movie(baseUrl, path[index+1], path[index+2])
+    elif type == 'gallery':
+        index = path.index('gallery')
+        scraped = scrape_gallery(baseUrl, path[index+1], path[index+2])
+    elif type == 'performer':
+        index = path.index('model')
+        scraped = scrape_model(baseUrl, path[index+1])
+    else:
+        return None
+  
+    return scraped
+
+def search(fragment, type):
+    if type == 'scene' or type == 'gallery':
+      name = re.sub(r'\W', '_', fragment['title']).upper()
+      date = fragment['date'].replace('-', '')
+  
+      scraper = globals()['scrape_' + ('movie' if type == 'scene' else type)]
+      res = scraper('https://metartnetwork.com', date, name)
+      if res != None:
+          return res
+
+def fetch(baseUrl, type, arguments):
+    url =f"{baseUrl}/api/{type}?{urlencode(arguments)}"
+    response = requests.get(url)
+    if response.status_code >= 400:
+        return None
+  
+    data = response.json()
+    return data
+
+def scrape_model(baseUrl, name):
+    data = fetch(baseUrl, 'model', {'name': name})
+    if data == None:
+        return None
+  
+    return map_model(baseUrl, data)
+
+def map_media(data, studio, baseUrl):
+    studioUrl = studio[1]
+  
+    return {
+        'Title': data['name'],
+        'Details': data['description'],
+        'URL': f"https://www.{studioUrl}{data['path']}",
+        'Date': data['publishedAt'][0:data['publishedAt'].find('T')],
+        'Tags': list(map(lambda t: {'Name': t}, data['tags'])),
+        'Performers': list(map(lambda m: map_model(baseUrl, m), data['models'])),
+        'Studio': {'Name': studio[0]}
+    }
+
+def scrape_movie(baseUrl, date, name):
+    data = fetch(baseUrl, 'movie', {'name':name, 'date': date})
+    if data == None:
+        return None
+
+    studio = get_studio(data['media']['siteUUID'])
+    res = map_media(data, studio, baseUrl)
+    res['Image'] = f"https://www.{studio[1]}{data['splashImagePath'] if 'splashImagePath' in data else data['coverCleanImagePath'] if 'coverCleanImagePath' in data else data['coverImagePath']}"
+
+    return res
+
+def scrape_gallery(baseUrl, date, name):
+    data = fetch(baseUrl, 'gallery', {'name':name, 'date': date})
+    if data == None:
+        return None
+
+    studio = get_studio(data['siteUUID'])
+    return map_media(data, studio, baseUrl)
+
+def map_model(baseUrl, model):
+    return {
+        'Name': model['name'],
+        'Gender': model['gender'].capitalize(),
```

Comment:
This should be `upper()` not capitalize
For the perforer gender enums all allcaps https://github.com/stashapp/stash/blob/ecac7a80137d830900bcd5fbe120c5c8ee648036/graphql/schema/types/performer.graphql#L1